test/run-tests

```diff
@@ -657,6 +658,26 @@ def get_status_context(variant, image_name, ansible_id):
     return f"{image_name}/{ansible_id}{suffix}"
 
 
+def is_role_distro_supported(image, repo_dir):
+    """
+    Return True if image is supported. Otherwise False.
+    """
+    distro = image["name"].split("-")[0]
+    version = image["name"].split("-")[1]
+    if version == 'x':
+        image_filename=(
+            image['source'].rsplit('/', 1)[1]
```

Comment:
How would this work with https://github.com/linux-system-roles/test-harness/commit/95372553cc0dad26ac045cd2a3583015c0d73510 if there isn't a `image['source']` field?  I suppose for now it is ok since the only `-x` version we have is with rhel-x - but please consider how this would work if there is `image['compose']` and no `image['source']`

Reply:
Good point. I can extract the `get_url()` function from the Task class or simply get the first numbers in compose URL. The second option is not so unique.

Reply:
Can't you just call `get_url()`? It would require to move the check after Task is created, would it be a problem?

Reply:
Fixed. I use `get_url()`. The function `is_role_distro_supported()` is a part of the Task class.